src/Visualization.jl

```diff
@@ -1,5 +1,5 @@
 using Graphs: SimpleGraph, Edge, edges, ne, nv
-using GraphMakie: graphplot!, GraphPlot, to_colormap, get_node_plot
+using GraphMakie: graphplot!, GraphPlot, to_colormap, get_node_plot, add_edge!, add_vertex!
```

Comment:
`add_edge!` and `add_vertex!` are part of the `Graphs` package, not `GraphMakie`

Reply:
True, fixed.